less/circle-background.less

```diff
@@ -0,0 +1,164 @@
+
+.circle-bg {
+  position: relative;
+  z-index: 1;
+  &:before {
+    content: "";
+    width: @circle-bg-md-diameter;
+    padding-bottom: @circle-bg-md-diameter;
+    border-radius: 500rem;
+    position: absolute;
+    background: @gray-base;
+    z-index: -1;
+    top: @circle-bg-translate-to-center;
+    left: @circle-bg-translate-to-center;
+    transform: translate(-@circle-bg-translate-to-center, -@circle-bg-translate-to-center);
+  }
+}
+
+/*Sizes*/
+
+.circle-bg-xs:before {
+  .circle-bg-variant(@circle-bg-xs-diameter,@circle-bg-xs-diameter);
+}
+
+.circle-bg-sm:before {
+  .circle-bg-variant(@circle-bg-sm-diameter,@circle-bg-sm-diameter);
+}
+
+.circle-bg-lg:before {
+  .circle-bg-variant(@circle-bg-lg-diameter,@circle-bg-lg-diameter);
+}
+
+.circle-bg-xl:before {
+  .circle-bg-variant(@circle-bg-xl-diameter,@circle-bg-xl-diameter);
+}
+
+/*Positions (center). Uses -c sufix*/
+
+.circle-bg-top-c:before {
+  top: 0;
+}
+
+.circle-bg-right-c:before {
+  right: -@circle-bg-translate-to-center;
+  left: inherit;
+}
+
+.circle-bg-left-c:before {
+  left: 0;
+  transform: translate(-@circle-bg-translate-to-center, -@circle-bg-translate-to-center);
+}
+
+.circle-bg-bottom-c:before {
+  transform: translate(-@circle-bg-translate-to-center, @circle-bg-translate-to-center);
+  bottom: 0;
+  top: inherit;
+}
+/*Positions (box edge)*/
+.circle-bg-top:before {
+  transform: translate(-@circle-bg-translate-to-center, -@circle-bg-translate-to-edge);
+  top: 0;
+}
+
+.circle-bg-right:before {
+  transform: translate(@circle-bg-translate-to-edge, -@circle-bg-translate-to-center);
+  left: inherit;
+  right: 0;
+}
+
+.circle-bg-left:before {
+  left: 0;
+  transform: translate(-@circle-bg-translate-to-edge, -@circle-bg-translate-to-center);
+}
+
+.circle-bg-bottom:before {
+  top: inherit;
+  bottom: 0;
+  transform: translate(-@circle-bg-translate-to-center, @circle-bg-translate-to-edge);
+}
+/*Combinations*/
+.circle-bg-top.circle-bg-left:before {
+  transform: translate(-@circle-bg-translate-to-edge, -@circle-bg-translate-to-edge);
+}
+
+.circle-bg-top.circle-bg-right:before {
+  transform: translate(@circle-bg-translate-to-edge, -@circle-bg-translate-to-edge);
+}
+
+.circle-bg-bottom.circle-bg-left:before {
+  transform: translate(-@circle-bg-translate-to-edge, @circle-bg-translate-to-edge);
+}
+
+.circle-bg-bottom.circle-bg-right:before {
+  transform: translate(@circle-bg-translate-to-edge, @circle-bg-translate-to-edge);
+}
+
+/*Not simetric center for big headings*/
+
+.circle-bg-center:before {
+  transform: translate(-@circle-bg-translate-to-center-asymmetric, 0%);
+  left: 0;
+}
+
+.circle-bg-clip {
+  overflow: hidden;
+}
+
+.circle-bg-blue-l:before {
+  background: @brand-pale-blue;
+}
+
+.circle-bg-blue {
+  color: @brand-white;
+
+  &:before {
+    background: @brand-blue;
+  }
+}
+
+.circle-bg-blue-d {
+  color: @brand-white;
+
+  &:before {
+    background: @brand-navy-minus-20;
+  }
+}
+// Visualization
+
+.circle-bg-margin {
```

Comment:
This margins were created for visualization purposes, because the dimensions of the element is given by the content and not by the circle, so the circles collapsed.
I maintained the margins thinking that maybe could be useful is some cases to give those spaces to the circle element. 
I added a note at the bottom of the doc:
"There are available utility clases to control the vertical margin of the circle background element.
 .circle-bg-xs-margin 
.circle-bg-sm-margin 
.circle-bg-margin 
.circle-bg-lg-margin 
.circle-bg-xl-margin "